### ISSUE TITLE
made autocraft work with additionnal outputs

### ISSUE THREAD

**Issue by SireTeddy** [AppliedEnergistics/Applied-Energistics-2#7331]
### Describe the feature

Hello, i want add a function ,when they have a pattern with a multiple output ; we can autocraft to have the additionnal output, an not just for the 1 st output ( screen to best understand)

### Reasons why it should be considered

I request it, because i play on a modpack with modern industrialisation, and a lot of crafts have 2 or more output (for items or fluids), and when i want to add on my autocraft system, a multiple output craft, i need to add on primary output, the first item, and add another pattern with the same craft, but the 2nd output on primary.
links of 2 crafts or the same outputs (but need now for really have autocrafts of they) here 
https://photos.google.com/share/AF1QipMFDR3_7MNC39OumMNv-cunx-y8_Ce08lmiCfyjkdb7cHpdMXPiExjzyqOueD7XCg?key=RWpxemJud1k4QThfSlhpdDNZbjZpYjNiZUMxRFFn

**Comment by Notarin** [comment]
The current implementation suffices as a solution already, does it not?

**Comment by SireTeddy** [comment]
Yes but the diy solution take a lot of pattern for the same result... My screens are just for 2 output, but on the modpack, i have craft with 6/7/8 outputs differents, so all indepedents output needs 1 pattern per output ? Just add this function will good.

**Comment by Notarin** [comment]
I understand what you're saying, and yes, one recipe per product. Multiple products, multiple recipes.
